test/unit/host_status/build_status_test.rb

```diff
@@ -7,6 +7,10 @@ def setup
     @status.host = @host
   end
 
+  test 'is valid' do
```

Comment:
The bug came about because #get_status would create a new ConfigurationStatus and then it would get saved, which failed by default unless you filled in the reported_at field.  I wanted to ensure that calling .new would result in a valid object - so it exercises the before_validation.